### PR TITLE
Blob support services are available only on data nodes.

### PR DESCRIPTION
**Pull request by andreidan**

### CODE REVIEW

File: blob/src/main/java/io/crate/plugin/BlobPlugin.java [hunk 1]
```diff
@@ -65,6 +67,10 @@ public String description() {
 
     @Override
     public Collection<Module> createGuiceModules() {
+        if (settings.getAsBoolean(Node.NODE_DATA_SETTING.getKey(), false)) {
```
Comment (matriv) [inline blob/src/main/java/io/crate/plugin/BlobPlugin.java]: why not use:
```
if (Node.NODE_DATA_SETTING.get(settings)) {
    return Collections.emptyList();
}```
Which will give the default value (true) if it's not explicitily set.

**Comment by matriv** [comment]
@andreidan plz also check test failurs

**Comment by andreidan** [comment]
retest this please

**Comment by andreidan** [comment]
@mfussenegger yep, will drop it

**Pull request merged**